docs/packaging.rst

```diff
@@ -45,6 +45,11 @@ on either.
 Other Python libraries
 ~~~~~~~~~~~~~~~~~~~~~~
 
+Hypothesis has *mandatory* dependencies on the following libraries:
+
+* :pypi:`attrs`
+* :pypi:`coverage`
```

Comment:
[There is ](https://github.com/HypothesisWorks/hypothesis-python/blob/master/setup.py#L61-L62) also enum34 required if you're building Hypothesis for Python 2.x.

Reply:
Good spot, f fixed.